listenbrainz/listenstore/influx_listenstore.py

```diff
@@ -41,23 +41,21 @@ class InfluxListenStore(ListenStore):
 
     def __init__(self, conf):
         ListenStore.__init__(self, conf)
-        self.redis = Redis(host=conf['REDIS_HOST'], port=conf['REDIS_PORT'], decode_responses=True)
-        self.redis.ping()
         self.influx = InfluxDBClient(host=conf['INFLUX_HOST'], port=conf['INFLUX_PORT'], database=conf['INFLUX_DB_NAME'])
 
     def get_listen_count_for_user(self, user_name, need_exact=False):
         """Get the total number of listens for a user. The number of listens comes from
-           a redis cache unless an exact number is asked for.
+           brainzutils cache unless an exact number is asked for.
 
         Args:
             user_name: the user to get listens for
             need_exact: if True, get an exact number of listens directly from the ListenStore
         """
 
         if not need_exact:
-            # check if the user's listen count is already in redis
+            # check if the user's listen count is already in cache
             # if already present return it directly instead of calculating it again
-            count = self.redis.get(REDIS_INFLUX_USER_LISTEN_COUNT + user_name)
+            count = cache.get(REDIS_INFLUX_USER_LISTEN_COUNT + user_name, decode=False)
```

Comment:
I think a comment should be added documenting why `decode=False` is needed here now.